src/pybind/mgr/dashboard_v2/tests/test_tcmu_iscsi.py

```diff
@@ -0,0 +1,101 @@
+from cherrypy.test.helper import CPWebCase
+import cherrypy
+import mock
+
+from ..controllers.tcmu_iscsi import TcmuIscsi
+from .helper import ControllerTestCase
+
+mocked_servers = [{
+    'ceph_version': 'ceph version 13.0.0-5083- () mimic (dev)',
+    'hostname': 'ceph-dev',
+    'services': [{'id': 'a:b', 'type': 'tcmu-runner'}]
+}]
+
+mocked_metadata = {
+    'ceph_version': 'ceph version 13.0.0-5083- () mimic (dev)',
+    'pool_name': 'pool1',
+    'image_name': 'image1',
+    'image_id': '42',
+    'optimized_since': 100.0,
+}
+
+mocked_get_daemon_status = {
+    'lock_owner': 'true',
+}
+
+mocked_get_counter = {
+    'librbd-42-pool1-image1.lock_acquired_time': [[10000.0, 10000.0]],
+    'librbd-42-pool1-image1.rd': 43,
+    'librbd-42-pool1-image1.wr': 44,
+    'librbd-42-pool1-image1.rd_bytes': 45,
+    'librbd-42-pool1-image1.wr_bytes': 46,
+}
+
+mocked_get_rate = {
+    'librbd-42-pool1-image1.rd': 43,
+    'librbd-42-pool1-image1.wr': 44,
+    'librbd-42-pool1-image1.rd_bytes': 45,
+    'librbd-42-pool1-image1.wr_bytes': 46,
+}
+
+
+class TcmuIscsiControllerTest(ControllerTestCase, CPWebCase):
```

Comment:
This test should not extend from `ControllerTestCase`, in this test you should access the controller through cherrypy.test functions like `getPage` and others.

Reply:
I'd like to keep other `vstart.sh-tests` and this test as similar as possible. Tbh, the functionality provided by `ControllerTestCase` is better than `CPWebCase` and I'm very much in favour of using all goodies of `ControllerTestCase`. 

Sure, I could resurrect parts of `ControllerTestCase` from the day before yesterday, but for which reason? This test could suddenly behave differently than other tests, and I don't like that.